internals/secrethub/list.go

```diff
@@ -100,9 +100,7 @@ func (cmd *LsCommand) Run() error {
 	secretPath, err := cmd.path.ToSecretPath()
 	if err == nil {
 		versions, err := client.Secrets().Versions().ListWithoutData(secretPath.Value())
-		if api.IsErrNotFound(err) {
-			return ErrResourceNotFound(cmd.path)
-		} else if err != nil {
+		if err != nil {
```

Comment:
This check is now done on the client and an `errSecretNotFound` is returned.

Reply:
Do we want to pass the generic "resource not found" error instead of a "secret not found" error, similar to what was there in the remove command?

Reply:
Because, similar as the `remove` command, the `list` command doesn't know whether the provided path points to a secret. Therefore, if no secret can be found on the given path a more generic error should be returned.

Reply:
Good point. I'll close the PR as no changes are required, then